Src/BootCamp.Chapter/Player.cs

```diff
@@ -29,79 +30,142 @@ public class Player
         /// Player items. There can be multiple of items with same name.
         /// </summary>
         private Inventory _inventory;
+
         /// <summary>
         /// Needed only for the extra task.
         /// </summary>
         private Equipment _equipment;
 
+        public List<Item> Items { get; set; }
```

Comment:
oke,

I did that but then for example this test was failing : 

```
public void AddItem_Adds_New_Item_In_Inventory()
        {
            var item = new Item("Potion", 20, 0.2f);

            _player.AddItem(item);
            var items = _player.GetItems(item.GetName());

            using (new AssertionScope())
            {
                items.Should().Contain(item);
            }
        }

because Items was empty and all things were in Inventory which contains the items. 
```

Reply:
You are close, just rather than your player.AddItem() talking to a list on player, this should be talking to the inventory. It's close!

Reply:
Oke, that is what I  mean or we do misunderstood each other. 
I did   ` _inventory.AddItem(item) ` but then the tests are failing as I described

Reply:
doing `_inventory.AddItem(item)` in the players `AddItem()` was the correct thing to do. 

The test failing means something else was wrong. Perhaps the players `GetItems` was not working correctly. 

You should be able to debug the tests and follow the code through to see where it's not working.

2 main points of failure will be
1. `AddItem` did not actually add the item to the inventory
2. `GetItem` did not actually retrieve the item from the inventory

Reply:
oke, will change that and hopefullY I can find out where the problems is.

Reply:
I think that I found the culprit 

I have this : 
```
public void AddItem(Item item)
        {
            _inventory.AddItem(item); 
        }
```

which calls this : 
```
public void AddItem(Item item)
        {
            Items.Add(item);
        }
```  

which is void so it do not return anything. So the item is never added to inventory as far as now can see.

Reply:
but right now I see these annoying screens when try to debug the problem : 

https://cdn.discordapp.com/attachments/663436264376238119/688431604028407820/unknown.png

Reply:
The return type being void is okay. 

The Add method just needs to add the record into the list which Inventory's `Items.Add(item);` is doing correctly. 

When you're able to debug you should see that the Inventory.Items has records in.

This makes me think something is wrong with either Players `GetItems` or Inventories `GetItems` for the tests to be failing.

Never really seen that before and i'm able to debug your code fine :S so hopefully someone else can reply with an answer :D

Reply:
solved. I hope the right way now